src/services/scheduler.ts

```diff
@@ -173,7 +173,7 @@ export async function init() {
       const user = await MetadataRepo.getUserFromDbId(db.pgName);
       const message = `Failed to bootstrap db ${db.pgName} on startup. Reason: ${bootstrap.reason}`;
       logger.error(message);
-      logErrSentry(message, user?.id, user?.email, db.alias);
+      logErrSentry({message}, user?.id, user?.email, db.alias);
```

Comment:
```suggestion
      logErrSentry(message instanceof Object ? message : { message, }, user?.id, user?.email, db.alias);
```

Reply:
well this message is always a string, I can add some type safety in the `logErrSentry` signature